difido-reports-common/src/main/java/il/co/topq/difido/model/remote/ExecutionMetadataUpdateRequest.java

```diff
@@ -0,0 +1,39 @@
+package il.co.topq.difido.model.remote;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ExecutionMetadataUpdateRequest {
```

Comment:
If we change the request to use the existing one using query string (See the comment in the *Execution Resource* file), there is no need for this class and it can be removed.